ecs-agent/tmds/handlers/fault/v1/handlers/handlers.go

```diff
@@ -36,56 +49,255 @@ func FaultNetworkFaultPath(fault string) string {
 		utils.ConstructMuxVar(v4.EndpointContainerIDMuxName, utils.AnythingButSlashRegEx), fault)
 }
 
-// TODO
 func (h *FaultHandler) StartNetworkBlackholePort() func(http.ResponseWriter, *http.Request) {
 	return func(w http.ResponseWriter, r *http.Request) {
+		var request types.NetworkBlackholePortRequest
```

Comment:
Will the request be logged? If not I think we should log it.

Reply:
You're right, made this change before our offline discussion. Will make an update for this.

Reply:
Logging handler does not log request body and logs at DEBUG level. Given the sensitive nature of fault injection endpoints, I think we should log the request body at INFO level.

https://github.com/aws/amazon-ecs-agent/blob/master/ecs-agent/tmds/logging/logging_handler.go#L32-L35